gradle.properties

```diff
@@ -1,5 +1,5 @@
 ltrVersion = 1.5.8
-elasticsearchVersion = 8.9.2
+elasticsearchVersion = 8.11.1
 luceneVersion = 9.7.0
```

Comment:
Elasticsearch version 8.11.1 uses Lucene version 9.8.0.
Please change to `luceneVersion = 9.8.0`